sdks/python/examples/cancellation/test_cancellation.py

```diff
@@ -1,13 +1,15 @@
 import pytest
 
+from examples.cancellation.worker import wf
 from hatchet_sdk import Hatchet, Worker
 
 
 # requires scope module or higher for shared event loop
 @pytest.mark.asyncio(scope="session")
 @pytest.mark.parametrize("worker", ["cancellation"], indirect=True)
 async def test_run(hatchet: Hatchet, worker: Worker) -> None:
-    run = hatchet.admin.run_workflow("CancelWorkflow", {})
+    run = wf.run()
```

Comment:
nit: do we want to encourage naming all the wokflows wf>

Reply:
good call, I'll change these to be better named